test/Microsoft.AspNetCore.Razor.Test.Common/Language/RazorProjectEngineBuilderExtensions.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language.IntegrationTests;
+using Microsoft.AspNetCore.Razor.Language.Intermediate;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public static class RazorProjectEngineBuilderExtensions
+    {
+        public static RazorProjectEngineBuilder AddTagHelpers(this RazorProjectEngineBuilder builder, params TagHelperDescriptor[] tagHelpers)
+        {
+            return AddTagHelpers(builder, (IEnumerable<TagHelperDescriptor>)tagHelpers);
+        }
+
+        public static RazorProjectEngineBuilder AddTagHelpers(this RazorProjectEngineBuilder builder, IEnumerable<TagHelperDescriptor> tagHelpers)
+        {
+            var feature = (TestTagHelperFeature)builder.Features.OfType<ITagHelperFeature>().FirstOrDefault();
+            if (feature == null)
+            {
+                feature = new TestTagHelperFeature();
+                builder.Features.Add(feature);
+            }
+
+            feature.TagHelpers.AddRange(tagHelpers);
+            return builder;
+        }
+
+        public static RazorProjectEngineBuilder AddDefaultImports(this RazorProjectEngineBuilder builder, params RazorSourceDocument[] imports)
```

Comment:
What do you guys think about lifting AddDefaultImports(...) into the primary extensions (src) instead of just being a test extension?

Reply:
I think we should show some restraint for now. It's not a bad idea. But I don't want to do it if ALL extenders need to ignore it and create their own feature implementation.

Reply:
I went ahead and did it after looking at the codez. It seems simple enough.